### DISCUSSION TITLE
Pagination without next page URL

### DISCUSSION THREAD

**Discussion by ghost** [engram-design/FeedMe#1030] [Ideas]
I have been importing a feed that is paginated (50 items per page) but it doesn't have a next page url, I've asked the supplier to add this to the feed but they won't do it. They suggested that their clients fetch the pages until they get a response with less than 50 or a 4xx. Would it be possible to, instead of selecting a pagination URL, set a page item limit so Feed Me would would fetch the next page until a page has less than the limit? They do supply a lastPageIndex value so that could be the number of loops Feed Me should do.

If not that would be a bummer but do let me know so I can start the search for a new plugin that does support this. 

Many thanks!

**Comment by darylknight** [comment]
I'd love to see an answer to this too. I'm importing from the Steam API, and their feed URLs look like this:

https://store.steampowered.com/wishlist/profiles/76561197999920602/wishlistdata/?p=0
https://store.steampowered.com/wishlist/profiles/76561197999920602/wishlistdata/?p=1
https://store.steampowered.com/wishlist/profiles/76561197999920602/wishlistdata/?p=2

There is no pagination URL within the page itself. Perhaps some kind of number could be added, or a "feed pagination format" setting so Feed Me could paginate through this structure?

Right now I'm rather clunkily just running three separate feeds on the three URLs above.
